packages/types/src/client.ts

```diff
@@ -69,6 +69,13 @@ export interface Client<O extends ClientOptions = ClientOptions> {
   /** Returns the current options. */
   getOptions(): O;
 
+  /**
+   * @inheritdoc
+   *
+   * TODO (v8): Make this a required method.
+   */
+  getSdkMetadata?(): O['_metadata'];
```

Comment:
l: Is there any reason not to make this type `SdkMetadata` right away? I feel like we generally and recently steered away from using type indirections like these because they tend to get a bit intransparent.

Reply:
Nope, no particular reason, will change it!

Reply:
[9082753](https://github.com/getsentry/sentry-javascript/pull/6643/commits/9082753e8ea276dc3f59d45f9af4a9041322ab43)